eclipse-scout-core/src/form/fields/smartfield/TreeProposalChooser.ts

```diff
@@ -215,6 +215,18 @@ export class TreeProposalChooser<TValue> extends ProposalChooser<TValue, Tree, P
     return leafs;
   }
 
+  /**
+   * This function creates a list of flat tree nodes from a list of lookup rows.
```

Comment:
```suggestion
   * This function creates a flat list of tree nodes from a list of lookup rows.
```

Reply:
This commit should not be in this PR.... :/